MapboxCoreNavigation/EventDetails.swift

```diff
@@ -365,42 +365,18 @@ extension UIApplicationState: Encodable {
 
 extension AVAudioSession {
     var audioType: String {
-        if isOutputBluetooth() {
+        if currentRoute.outputs.contains(where: { [.bluetoothA2DP, .bluetoothHFP, .bluetoothLE].contains($0.portType) }) {
             return "bluetooth"
         }
-        if isOutputHeadphones() {
+        if currentRoute.outputs.contains(where: { [.headphones, .airPlay, .HDMI, .lineOut, .carAudio, .usbAudio].contains($0.portType) }) {
             return "headphones"
         }
-        if isOutputSpeaker() {
+        if currentRoute.outputs.contains(where: { [.builtInSpeaker, .builtInReceiver].contains($0.portType) }) {
             return "speaker"
         }
+//        if currentRoute.outputs.contains(where: { [.builtInMic, .headsetMic, .lineIn].contains($0.portType) }) {
```

Comment:
I added `bluetoothHFP` to the Bluetooth cases above and `carAudio` and `usbAudio` to the headphone cases, but I wasn’t sure what to do about these microphone-related types, so they continue to be classified as “unknown”.

Reply:
Falling back to unknown sounds reasonable. Let's drop this commented code.

Reply:
Removed in #2059.